src/client/src/components/ActionBar/StatusFilter.jsx

```diff
@@ -0,0 +1,28 @@
+import { Dropdown } from "monday-ui-react-core";
+import React from "react";
+
+const StatusFilter = ({ filter, setFilter }) => {
```

Comment:
note that you're not using the "filter" prop and moving it to the DropDown, that's why even though you've set "all" to be the default it doesn't work, also changing this prop will affect nothing.